update_gas_prices.py

```diff
@@ -64,26 +38,26 @@
     # Prepare the dictionaire
     dict_prices = {
         PREVIOUS_WEEK: current_data[CURRENT_WEEK],
-        CURRENT_WEEK: {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: {}},
+        CURRENT_WEEK: {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: gas_info},
     }
+
     # Parse the data
-    i += 1
-    while i < len(gas_info) - 1:
-        gas = gas_info[i]
+    for key, value in gas_info.items():
         price = None
         # skip to next line in case fuelPrice is not a float
         while price is None:
             try:
-                price = float(gas_info[i + 1][:-1].replace(",", "."))
+                price = float(value.replace(",", "."))
```

Comment:
I suggest replacing this line with
```python 
price = f"{float(value.replace(',', '.')):.3f}"
```

The idea is to format the number to always display three decimal places. For example, `1.7` will become `1.700`.

Reply:
We can't assign the price value here as a string because we need, after that, to make a sum and get Gasolina IO98 price.

What we can do is an auxiliary function to transform all prices to three decimal digits before the tweet and historical update. I'll create an issue to do that.